conanfile.py

```diff
@@ -1,38 +1,57 @@
-from conans import ConanFile, tools
+from os.path import join
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+
+VERSION = "2.5.0.post1"
+BUILD_SUBFOLDER = "tensorflow/lite/tools/make"
 
-VERSION = "2.1.0.post1"
 
 class TFLiteConan(ConanFile):
     # Basic info
     name = "tensorflow-lite"
     version = VERSION
     scm = dict(type="git",
                url="https://github.com/tensorflow/tensorflow.git",
-               revision=f"d855adfc5a0195788bf5f92c3c7352e638aa1109")
+               revision=f"f394a768719a55b5c351ed1ecab2ec6f16f99dd4")
 
     # Other package details
     description = "https://www.tensorflow.org"
     license = "Apache-2.0"
 
     # Conan build process settings
-    exports = ["01-fix-missing-sources.patch", "02-tar-instead-of-zip.patch"]
+    exports = ["01-tar-instead-of-zip.patch", "02-ignore-nnapi.patch"]
     settings = "os", "arch", "compiler", "build_type"
 
-    build_subfolder = "tensorflow/lite/tools/make"
+    # def system_requirements(self):
+    #     installer = tools.SystemPackageTool()
+    #     installer.install("make")
 
     def requirements(self):
-        self.requires('flatbuffers/1.11.0@google/stable')
+        self.requires('flatbuffers/1.12.0@google/stable')
 
     def source(self):
-        tools.patch(patch_file="01-fix-missing-sources.patch")
-        tools.patch(patch_file="02-tar-instead-of-zip.patch")
-        self.run(f"{self.build_subfolder}/download_dependencies.sh")
+        tools.patch(patch_file="01-tar-instead-of-zip.patch")
+        tools.patch(patch_file="02-ignore-nnapi.patch")
+
+        self.run(join(BUILD_SUBFOLDER, "download_dependencies.sh"))
 
     def build(self):
-        if self.settings.arch == "armv8":
-            self.run(f"{self.source_folder}/{self.build_subfolder}/build_aarch64_lib.sh")
-        else:
-            self.run(f"{self.source_folder}/{self.build_subfolder}/build_lib.sh")
+        build_folder = join(self.source_folder, BUILD_SUBFOLDER)
+        is_aarch64 = self.settings.arch == "armv8"
+
+        # Note: the only advantage of using scripts is that when building
+        #  on aarch64 with low memory, building will be single-threaded
+
+        # if is_aarch64:
+        #     self.run(join(build_folder, "build_aarch64_lib.sh"))
+        # else:
+        #     self.run(join(build_folder, "build_lib.sh"))
+
+        target_opt = "TARGET=aarch64" if is_aarch64 else ""
+        env_vars = {'SCRIPT_DIR': build_folder, 'TENSORFLOW_DIR': self.source_folder}
+
+        autotools = AutoToolsBuildEnvironment(self)
+        with tools.chdir(self.source_folder):
+            autotools.make(target=f"{target_opt} -f {build_folder}/Makefile micro", vars=env_vars)
```

Comment:
Added this change to gain more control over the Makefile.
The build scripts are basically just forwarded calls with args, and by default `all` is built.
Had to build the target `micro` specifically (only `libtensorflow-lite.a` basically) because on OSX the other binaries fail to build.